ktlint-ruleset-standard/src/main/kotlin/com/github/shyiko/ktlint/ruleset/standard/SpacingAroundParensRule.kt

```diff
@@ -4,13 +4,19 @@ import com.github.shyiko.ktlint.core.Rule
 import com.github.shyiko.ktlint.core.ast.ElementType.IDENTIFIER
 import com.github.shyiko.ktlint.core.ast.ElementType.LPAR
 import com.github.shyiko.ktlint.core.ast.ElementType.RPAR
+import com.github.shyiko.ktlint.core.ast.ElementType.SUPER_KEYWORD
 import com.github.shyiko.ktlint.core.ast.ElementType.VALUE_ARGUMENT_LIST
 import com.github.shyiko.ktlint.core.ast.ElementType.VALUE_PARAMETER_LIST
 import com.github.shyiko.ktlint.core.ast.nextLeaf
 import com.github.shyiko.ktlint.core.ast.prevLeaf
 import org.jetbrains.kotlin.com.intellij.lang.ASTNode
 import org.jetbrains.kotlin.com.intellij.psi.PsiWhiteSpace
 
+/**
+ * Ensures there are no extra spaces around parentheses.
+ *
+ * See https://kotlinlang.org/docs/reference/coding-conventions.html, "Horizontal Whitespace"
```

Comment:
if you want, you can use the fragment: https://kotlinlang.org/docs/reference/coding-conventions.html#horizontal-whitespace

Reply:
ah yes I was looking for that and just looked through it. will update.